app/uk/gov/hmrc/testuser/models/TestUser.scala

```diff
@@ -39,14 +40,40 @@ case class TestOrganisation(override val username: String,
                             vrn: Vrn,
                             override val _id: BSONObjectID = BSONObjectID.generate) extends TestUser
 
-case class TestIndividualResponse(username: String, password: String, saUtr: SaUtr, nino: Nino)
-case class TestOrganisationResponse(username: String, password: String, saUtr: SaUtr, empRef: EmpRef, ctUtr: CtUtr, vrn: Vrn)
+case class CreateTestIndividualResponse(username: String, password: String, saUtr: SaUtr, nino: Nino)
+case class CreateTestOrganisationResponse(username: String, password: String, saUtr: SaUtr, empRef: EmpRef, ctUtr: CtUtr, vrn: Vrn)
```

Comment:
We need extra case classes.
I will use 
- TestOrganisationCreatedResponse
- TestIndividualCreatedResponse